afisha-server/server/api/controllers/eventController.js

```diff
@@ -1,4 +1,6 @@
 const Event = require('../models/eventModel');
+const ErrorBuilder = require('../errors/errorBuilder');
+const ERROR_LIST = require('../errors/errorList');
 
 const eventController = {
     findEventById: (req, res, next, id) => {
```

Comment:
'next' and 'id' is redundant in this case.

Reply:
@doslechescake I pass the event with next in req.event so that I can use it for delete and update methods.